lib/commands/doctor/checks/logged-in-user-owner.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const os = require('os');
+const path = require('path');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+const fs = require('fs');
+const ghostUser = require('../../../utils/use-ghost-user');
+
+const taskTitle = 'Checking logged in user and directory owner';
+
+function loggedInUserOwner(ctx) {
+    const uid = process.getuid();
+    const gid = process.getgroups();
+    let ghostStats;
+    const dirStats = fs.lstatSync(path.join(process.cwd()));
+    const contentDirStats = fs.lstatSync(path.join(process.cwd(), 'content'));
+
+    try {
+        ghostStats = ghostUser.getGhostUid();
+    } catch (e) {
+        throw e;
+    }
+
+    // CASE 1: check if ghost user exists and if it's currently used
+    if (ghostStats && ghostStats.uid && ghostStats.uid === uid) {
```

Comment:
Not 100% on adding this check....seems like this would be better moved to a systemd extension check (or at least a separate check)

Reply:
> Not 100% on adding this check

Can you explain this to me a bit further? This check is trying to prevent that a user is running commands as a logged in `ghost` user.

Reply:
So the idea behind doctor checks is to have one thing per check, right? Checking whether or not the current user is `ghost` is something entirely different from checking whether or not the current directory is owned by the current running user. It complicates the `enabled`/`skip` checks as well, because for the local process manager, it's good to check whether or not the current user owns the current folder, but for the local process manager the user _can_ be ghost.

Reply:
Alright. Gotcha! 👍 I think I had a little false thinking in my brain, assuming that those checks belong together. I split them now in two separate checks and also skipped the `ghost-user` check when the local process manager is used.